database/factories/ProductImageFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use IndieHD\Velkart\ProductImage\ProductImage;
+use Illuminate\Http\UploadedFile;
+use IndieHD\Velkart\Product\Product;
+
+$factory->define(ProductImage::class, function (Faker\Generator $faker) {
+
+    $productId = factory(Product::class)->create();
```

Comment:
Doesn't `create()` return a model instance (not just the `id`)?

Reply:
Yea, I originally had it returning the id and refactored it and forgot to rename the property. I'll fix.

Reply:
Hmm yea its interesting that this passes and doesn't throw a PDOException ?